main/manager.c

```diff
@@ -712,6 +712,9 @@
 			<parameter name="OtherChannelId">
 				<para>Channel UniqueId to be set on the second local channel.</para>
 			</parameter>
+            <parameter name="Gosub">
```

Comment:
I think is better named PreDialGoSub to be more explicit about what it is.

Reply:
Cool, updated with your recommendation.